typings/externs.d.ts

```diff
@@ -4,43 +4,57 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-import _Crdp from "../node_modules/vscode-chrome-debug-core/lib/crdp/crdp";
+import _Crdp from '../node_modules/vscode-chrome-debug-core/lib/crdp/crdp';
 
 declare global {
-  module LH {
+  namespace LH {
```

Comment:
was this change necessary? I don't actually remember the reasoning (and it's possible the reasons no longer apply as this file changed), but I do remember it was a deliberate decision at the time :)

Reply:
no it keeps getting automatically changed or a bad rebase or something :)